cactusbot/handler.py

```diff
@@ -37,7 +74,31 @@ def __init__(self, *handlers):
         return result
 
     def translate(self, packet, handler):
-        """Translate handler responses to Packets."""
+        """Translate :obj:`Handler` responses to :obj:`Packet`.
+
+        Parameters
+        ----------
+        packet : :obj:`Packet`, :obj:`str`, :obj:`tuple`, :obj:`list`,
+                 :obj:`StopIteration` or :obj:`None`
+            The packet to turn the handler response into
+        handler : :obj:`Handler`
+            The handler response to turn into a packet
+
```

Comment:
This is really pretty! :heart_eyes: 

We need to mention what each thing *does*, though.

- `Packet` is immediately `yield`ed
- `str` is converted into a `text` field in a `MessagePacket`
- `tuple` or `list` is iterated over, yielding each item
- `StopIteration` stops future packets from being passed
- `None` is ignored